examples/memory_basic_usage.py

```diff
@@ -26,7 +26,10 @@ def expensive_computation(data, column_index=0):
     return data[column_index]
 
 
-data = np.random.randn(int(1e5), 10)
+# to hit the cache between Python session, be sure to set the random seed to
+# generate deterministic data.
```

Comment:
I would be even more explicit: "Indeed, if the data is not deterministic, the Memory will not be able to reuse the cache from one run the another"